helm/auditmanager/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: auditmanager
 description: A Helm chart for MOSIP Auditmanager module
 type: application
-version: 1.3.0-beta.1-develop
+version: 1.3.0-develop
 appVersion: ""
 dependencies:
   - name: common
```

Comment:
audit-manager/deploy
/install.sh

CHART_VERSION=1.3.0-beta.1-develop
change to 
CHART_VERSION=1.3.0-develop